jwql/jwql_monitors/monitor_mast.py

```diff
@@ -143,8 +143,13 @@ def instrument_keywords(instrument, caom=False):
         A DataFrame of the keywords
     """
     # Retrieve one dataset to get header keywords
+    if not caom:
```

Comment:
What does this logic do? I am just curious, what is special about this program?

Reply:
This block queries two different MAST services. Either Caom.Filtered or Jwst.Instrument.Filtered.<instrument>. The two services seem to use two different keywords for the program ID, so this just makes sure the correct keyword is used.